packages/create-config/index.js

```diff
@@ -1,16 +1,21 @@
 #!/usr/bin/env node
 import { execSync } from 'node:child_process';
 import { statSync } from 'node:fs';
+// biome-ignore lint/nursery/noRestrictedImports: ignore
+import path from 'node:path';
 
 const fileExists = filePath => {
   const stat = statSync(filePath, { throwIfNoEntry: false });
   return stat?.isFile();
 };
 
 const getPackageManager = () => {
-  if (fileExists('bun.lockb')) return 'bun';
-  if (fileExists('yarn.lock')) return 'yarn';
-  if (fileExists('pnpm-lock.yaml')) return 'pnpm';
+  // get the root of the repository
+  const repositoryRoot = execSync('git rev-parse --show-toplevel').toString().trim();
```

Comment:
If we're going to use this here, I guess we should also use it for `fileExists('pnpm-workspace.yaml')` and when running `execSync(cmd)`? If so, please either remove it or apply everywhere.

Reply:
I see where you are coming from, but i think it is ok as it is (from the perspective of pnpm at least).
Because i could intentionally execute this command only in a sub-package of a pnpm workspace:

```sh
cd ./my-repository/packages/my-project
pnpm create @knip/config
```
Which would add the dependency only to the `./my-repository/packages/my-project/package.json`.

Whereas this would add it to the monorepo root (`./my-repository/package.json`):

```sh
cd ./my-repository
pnpm create @knip/config
```